backend/models/research_collaboration.py

```
@@ -0,0 +1,24 @@
+from sqlalchemy import Table, Column, Integer, ForeignKey, String, Text
+from sqlalchemy.orm import relationship
+from database.session import Base
+
+# Many-to-Many table for research collaborators
+research_collaborators = Table(
+    "research_collaborators",
+    Base.metadata,
+    Column("research_id", Integer, ForeignKey("research_collaborations.id")),
+    Column("user_id", Integer, ForeignKey("users.id"))
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)